src/test/java/pl/javastart/streamstask/ExampleTest.java

```diff
@@ -1,6 +1,101 @@
 package pl.javastart.streamstask;
 
+import org.assertj.core.api.SoftAssertions;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
 public class ExampleTest {
+    List<User> users = TestData.getUsers();
+    List<Expense> expenses = TestData.getExpenses();
```

Comment:
Wrzuć to do metody `@BeforeEach`, żeby zagwarantować, że potencjalne side effecty nie będą miały wpływu na pozostałe testy